src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -91,12 +120,27 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   return (
     <>
       <div className={containerClasses} onClick={onEventBoxClick}>
+        <button className={expandClasses}>
+          <FontAwesomeIcon
+            icon={regularSquare}
+            className="ScheduleEvent__expand--square"
+          />
+          <FontAwesomeIcon
+            icon={solidExpand}
+            className="ScheduleEvent__expand--arrows"
+          />
+          <FontAwesomeIcon
+            icon={solidCompress}
+            className="ScheduleEvent__expand--arrows--out"
+          />
+        </button>
+
         <div className="ScheduleEvent__info">
           <div className="ScheduleEvent__title">{event.name}</div>
           <div className="ScheduleEvent__host">by {event.host}</div>
         </div>
 
-        <div className={ScheduleEventBookmarkClass} onClick={bookmarkEvent}>
+        <div className="ScheduleEvent__bookmark" onClick={bookmarkEvent}>
```

Comment:
why was that changed?

Reply:
there were no more modifiers for this class name